htlcswitch/link_test.go

```diff
@@ -1025,9 +1020,8 @@ func TestChannelLinkMultiHopInsufficientPayment(t *testing.T) {
 	).Wait(30 * time.Second)
 	if err == nil {
 		t.Fatal("error haven't been received")
-	} else if !strings.Contains(err.Error(), "insufficient capacity") {
```

Comment:
It may still be useful to report the error string we give out, for the sake of the RPC client. So we'd also extend this new `assertFailureCode` method to still assert our internal string.

Reply:
This sounds like a test that does not belong here. Is it right that you want tests that ensure that we return a specific string for an error over the rpc interface? In that case I'd think we should add those to `lnwire`. But how relevant is this really? We move to structured errors for `SendToRoute`, so for that call these strings won't appear anymore. Also `SendPayment` is moving to higher level structured messages.